index.html

```
@@ -0,0 +1,178 @@
+
+        <div id="drop-zone" class="drop-zone">
+          <div class="drop-icon" aria-hidden="true">↓</div>
+          <strong>Drop files or folders here</strong>
+          <span>or use one of the local pickers</span>
+          <div class="picker-row">
+            <button id="pick-files" class="button secondary" type="button">Choose files</button>
+            <button id="pick-folder" class="button secondary" type="button">Choose folder</button>
+          </div>
+        </div>
+
+        <input id="file-input" class="sr-only" type="file" multiple />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)